samples/cpp/tutorial_code/ml/neural_network/neural_network.cpp

```diff
@@ -0,0 +1,58 @@
+#include <opencv2/ml/ml.hpp>
+
+using namespace std;
+using namespace cv;
+using namespace cv::ml;
+
+int main(int argc, char* argv[])
```

Comment:
Thanks for your comment, I already guessed that this might be the reason; I already made a new commit. :+1: